### DISCUSSION TITLE
Spotify Integration vs. Add-On

### DISCUSSION THREAD

**Discussion by wilwizard1979** [hassio-addons/addon-spotify-connect#220] [Q&A] [answered]
I have just realized that I have the Spotify integration but not the add-on installed.
The integration is working fairly well.
What is the difference between the Spotify Integration and the Spotify Connect Add-on?

**Comment by frenck** [comment, marked as answer]
The Spotify integration integrates Home Assistant with Spotify.

The add-on is an Spotify Connect end-point which you can use with Spotify.
